src/components/shop/confirm.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @module components/shop
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+
+import { setVisible, resize, getHalfRectBounds, getInnerRectBounds, createRect } from "./shop-layout.js";
+import { createConfirmButtons } from "./menu-buttons.js";
+
+export const createConfirm = (scene, config, bounds) => {
+    const { buttonsRight } = config.menu;
+    const { styleDefaults } = config;
+
+    const confirmContainer = scene.add.container();
+    const innerBounds = getOffsetBounds(bounds, getInnerRectBounds(bounds, buttonsRight));
+    const yOffset = bounds.height / 2 + bounds.y;
+
+    confirmContainer.buttons = createConfirmButtons(scene, innerBounds, config, yOffset); // hard coded value nono
+
+    confirmContainer.add([
+        createRect(scene, getHalfRectBounds(bounds, !buttonsRight), 0xff0000),
+        createRect(scene, getHalfRectBounds(bounds, buttonsRight), 0xff00ff),
+        createRect(scene, innerBounds, 0x0000ff),
+        scene.add.text(innerBounds.x, promptY(bounds), config.confirm.prompts.buy, styleDefaults).setOrigin(0.5),
+        scene.add.text(innerBounds.x + 20, currencyY(bounds), "1000", styleDefaults).setOrigin(0.5),
+        scene.add.image(innerBounds.x - 20, currencyY(bounds), "shop.balanceIcon"),
```

Comment:
Possibly using "shop.balanceIcon" might be a bad idea - e.g: there could be multiple shops and they'll be named whatever they are called in `main,js`.

If you need to merge any of this now though just make a note we need to cover these options before release.

Reply:
Yeah, this should be pulling a value from config- fixed.